src/components/Card.js

```diff
@@ -8,9 +8,31 @@ import {
   Text,
   Stack,
 } from '@chakra-ui/core';
-
+import URL from 'url-parse';
```

Comment:
Is there any way we can rely solely on the URL API? https://developer.mozilla.org/en-US/docs/Web/API/URL - the browser support is pretty much in line with what we need, and would allow us to eliminate the need for a dependency where we can!

I'm totally open to discussing this, if leveraging the URL API has some ramification I'm not thinking of that the `url-parse` package helps us solve, I'm all for it. Just figured I'd throw it out there!

Reply:
Honestly, just never used that API and just straight for npm, glad you brought that up. I'll swap it over.